x/issuance/handler.go

```diff
@@ -11,106 +11,28 @@ import (
 // NewHandler creates an sdk.Handler for issuance messages
 func NewHandler(k keeper.Keeper) sdk.Handler {
 	return func(ctx sdk.Context, msg sdk.Msg) (*sdk.Result, error) {
+		msgServer := keeper.NewMsgServerImpl(k)
+
 		ctx = ctx.WithEventManager(sdk.NewEventManager())
 
 		switch msg := msg.(type) {
-		case types.MsgIssueTokens:
-			return handleMsgIssueTokens(ctx, k, msg)
-		case types.MsgRedeemTokens:
-			return handleMsgRedeemTokens(ctx, k, msg)
-		case types.MsgBlockAddress:
-			return handleMsgBlockAddress(ctx, k, msg)
-		case types.MsgUnblockAddress:
-			return handleMsgUnblockAddress(ctx, k, msg)
-		case types.MsgSetPauseStatus:
-			return handleMsgSetPauseStatus(ctx, k, msg)
+		case *types.MsgIssueTokens:
+			res, err := msgServer.IssueTokens(sdk.WrapSDKContext(ctx), msg)
+			return sdk.WrapServiceResult(ctx, res, err)
+		case *types.MsgRedeemTokens:
+			res, err := msgServer.RedeemTokens(sdk.WrapSDKContext(ctx), msg)
+			return sdk.WrapServiceResult(ctx, res, err)
+		case *types.MsgBlockAddress:
+			res, err := msgServer.BlockAddress(sdk.WrapSDKContext(ctx), msg)
+			return sdk.WrapServiceResult(ctx, res, err)
+		case *types.MsgUnblockAddress:
+			res, err := msgServer.UnblockAddress(sdk.WrapSDKContext(ctx), msg)
+			return sdk.WrapServiceResult(ctx, res, err)
+		case *types.MsgSetPauseStatus:
+			res, err := msgServer.SetPauseStatus(sdk.WrapSDKContext(ctx), msg)
+			return sdk.WrapServiceResult(ctx, res, err)
 		default:
 			return nil, sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "unrecognized %s message type: %T", types.ModuleName, msg)
 		}
 	}
 }
```

Comment:
We can delete handlers. Looks like they aren't used anymore. There's a note in confluence with more info.

Reply:
Removed in https://github.com/Kava-Labs/kava/pull/1033/commits/3eca2a77b882ff873bc81155f5b1ef4870084f4c. For the AppModule's Route() method I just passed `nil` as the sdk.Handler.